tests/main.rs

```diff
@@ -7,6 +7,8 @@ mod mouse {
     fn move_and_press() {
         let mouse = Mouse::new();
         mouse.move_to(500, 500);
+        let pos = mouse.get_position().unwrap();
+        assert!(pos.x == 500);
```

Comment:
Sorry forgot to add this to the review

Reply:
Tested on Linux and Windows, yes.
Was that the feedback/question?

Reply:
Yeah and that assert macro, thats all

Reply:
Do you want me to remove the `assert!`? 
I've confirmed the `assert!` passes, if that's the question.

Reply:
yeah, remove the assert and its all good 😄

Reply:
I've removed the `assert!` as requested.
Just a note, in general, tests in Rust typically have `assert!` macros to ensure pass/fail of the functionality they are testing.